practice/pyramid/site.css

```diff
@@ -0,0 +1,38 @@
+
+.row {
+  display: flex;
+  border: 1px solid black;
+  margin: 20px;
+  height: 100px;
```

Comment:
Although it works correctly here, I suggest setting widths from the top down and heights from the bottom up, so having no height on `.row` and instead having `height: 80px` on the inner boxes. If you do end up specifying multiple heights, they can sometimes conflict and not all block elements expand to take up the height of the parent by default.